cli/rbd/trash/trash.py

```diff
@@ -0,0 +1,240 @@
+from copy import deepcopy
+
+from cli import Cli
+from cli.utilities.utils import build_cmd_from_args
+
+# from .trash_purge_schedule import TrashPurgeSchedule
+
+
+class Trash(Cli):
+    """
+    This module provides CLI interface to manage snapshots from images in pool.
+    """
+
+    def __init__(self, nodes, base_cmd):
+        super(Trash, self).__init__(nodes)
+        self.base_cmd = base_cmd + " trash"
+        # self.trash_purge_schedule = TrashPurgeSchedule(nodes, self.base_cmd)
```

Comment:
same here.

Reply:
This will be required in my subsequent PRs for trash. like I said this is just a partial PR. So I would like to retain this here for now. So I don't miss out on adding it in the next PR